Ninano/Ninano/Notice/InterestLisViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  InterestLisViewController.swift
+//  Ninano
+//
+//  Created by KYUBO A. SHIM on 2022/07/14.
+//
+
+import UIKit
+
+class InterestLisViewController: UIViewController {
```

Comment:
```suggestion
final class InterestLisViewController: UIViewController {
```

더이상 상속되지 않는 것을 의미하는 final을 붙이는게 좋아보여요 ~